practice/src/component/Button/Button.tsx

```diff
@@ -0,0 +1,47 @@
+import './button.css';
+
+interface ButtonProps {
+  /**
+   * Is this the principal call to action on the page?
+   */
+  primary?: boolean;
+  /**
+   * What background color to use
+   */
+  backgroundColor?: string;
+  /**
+   * How large should the button be?
+   */
+  size?: 'small' | 'medium' | 'large';
+  /**
+   * Button contents
+   */
+  label: string;
```

Comment:
the required field should be on top and the optional fields should be bottom, you can also sort them alphabetly